db/schema.rb

```diff
@@ -11,17 +11,25 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160913011553) do
+ActiveRecord::Schema.define(version: 20161206210706) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+  enable_extension "pg_stat_statements"
```

Comment:
Might be related to your Postgres version? It's probably save to just delete them